src/public/app/widgets/title_bar_buttons.js

```diff
@@ -30,9 +30,13 @@ const TPL = `
         height: 40px;
         width: 40px;
     }
+    .title-bar-buttons .top-btn.active{
+        background-color:var(--accented-background-color);
+    }
     </style>
 
     <!-- divs act as a hitbox for the buttons, making them clickable on corners -->
+    <div class="top-btn" title="Bring this window to the top. "><button class="btn bx bx-pin"></button></div>
```

Comment:
It doesn't just bring it on top, it keeps it always on top, right?

Reply:
yes

Reply:
That was a hint that the title should reflect this ;)